introduction.md

```diff
@@ -1,2 +1,6 @@
-Hi I'd like to introduce myself. I'm Derick
-Ruby, JavaScript, HTML & CSS, iOS or Electives
+Hi I'd like to introduce myself. I'm Derick.
+
+* Ruby
+* JavaScript
+* HTML
+* CSS
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?